posts/2015-04-20-mix-it-2015-pt1.md

```diff
@@ -0,0 +1,50 @@
+Intro
+
+## L'importance de la performance :
+
+[Paul Bakaus](https://twitter.com/pbakaus) a insisté sur l'importance des performances d'une application :
+
+**Temps de chargement : 1s**
+
+Votre application doit être prête à l'emploi en maximum 1 seconde dès le premier accès de l'utilisateur. Sinon vous commencez à perdre des utilisateurs. À partir de 3 secondes, vous perdez 40% des utilisateurs. Et au-delà de 10 secondes, il ne reste plus personne !
+
+**Temps de réponse : 100ms**
+
+Lorsque votre utilisateur effectue une action, il doit recevoir un retour en moins de 100ms. Ça ne veut pas dire que la requête, par exemple, doit être traitée par votre serveur dans ce laps de temps, mais un loader doit s'afficher ou le bouton changer d'état. Note : l'utilisateur commence à ressentir un "retard" à partir de 150ms et une réelle gêne au-delà de 200ms.
+
+**Boucle et animation : 6ms**
+
+Vos animations et transitions doivent tourner à une fréquence de 60 images par seconde pour une impression de fluidité. Cela donne une période (temps disponible pour génerer chaque image) de 16ms. Sur ces 16ms, il est recommandé de n'en utiliser que 6 au maximum pour garantir 60 FPS de manière constante. Pourquoi ? On réserve 4ms pour les traitements intégrés (gpu, navigateur, etc...), ce qui nous laisse donc 12ms. Si on s'autorise à utiliser entièrement ces 12ms, la moindre frame dépassant ce temps d'exécution entraînera un dépassement. En ne s'autorisant que la moitié de ces 12ms (6ms), on prévoit le cas des frames trop longues !
+
+[Plus de détails ici](https://docs.google.com/document/d/1bYMyE6NdiAupuwl7pWQfB-vOZBPSsXCv57hljLDMV8E)
+
+
+## Le succès Lichess.org
+
+[Lichess.org](http://fr.lichess.org/) est un site de jeu d'échecs en ligne.
+
+Le projet est open-source, gratuit à 100% pour tous ses utilisateurs et sans publicité !
```

Comment:
`Open Source` doesn't have a hyphen in french

http://fr.wikipedia.org/wiki/Open_source